src/ReduxFormContext.js

```diff
@@ -2,17 +2,21 @@ import * as React from 'react'
 
 export const ReduxFormContext = React.createContext(null)
 
+export const renderChildren = (
```

Comment:
@Viktor-the-great  Is it necessary to pull out the renderChildren function to fix the issue? Wouldn't just changing the order of 
```
            _reduxForm,
            ref: forwardedRef,
            ...rest
```
to 
```
    ...rest,
    _reduxForm,
    ref: forwardedRef
```
in the inline function still fix it? 

Just wondering. Thanks!

Reply:
just wanted to add a test